app/jobs/destroy_person_job.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class DestroyPersonJob < ApplicationJob
+  queue_as :default
+
+  def perform(id)
+    person(id)&.destroy
```

Comment:
Shouldn't this be `.destroy!`? So that the job raises an error which is reported to appsignal and then the job will try again in case destroying fails?

```suggestion
    person(id)&.destroy
```